src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -8,11 +8,44 @@
 
   <script>
     @if ($crud->getPersistentTable())
```

Comment:
Since the persistentTable functionality has gotten pretty big, maybe it would make sense to move it into a separate blade file, like we have for the export buttons? Otherwise it kind of feels like it bloats the ```datatables_logic``` file now.